core/browser_events.ts

```diff
@@ -181,16 +184,16 @@ export function bind(
  * @return The function call.
  * @alias Blockly.browserEvents.unbind
  */
-export function unbind(bindData: Data): Function {
-  let func;
+export function unbind(bindData: Data): (e: Event) => void {
+  let callback = bindData[bindData.length - 1][2];
```

Comment:
This seems potentially unsafe if bindData is empty?

Reply:
It is, but that should never happen b/c people *should* only pass data provided by `bind` or `conditionalBind` to this function.

I think our options are (1) we continue assuming this will be fine, or (2) we modify the return type of the function to be nullable.